Src/LiquidProjections.NHibernate/ExceptionPolicy.cs

```diff
@@ -0,0 +1,9 @@
+namespace LiquidProjections.NHibernate
+{
+    /// <summary>
+    /// A delegate that can be implemented to retry projecting a batch of transactions when it fails.
+    /// </summary>
+    /// <param name="exception">The exception that occured that caused this batch to fail.</param>
+    /// <param name="count">The number of times that this batch has failed.</param>
+    public delegate bool RetryPolicy(ProjectionException exception, int count);

```

Comment:
A delegate is usually called the same way as a method, e.g. `RetryProjectionAttempt` or something like that.
The `count` represents probably the number of attempts...?